app/index.js

```diff
@@ -35,7 +32,7 @@ const osLibraries = [
 
 // Grab a list of all CF components, we'll use it later.
 const components = requestPromise( {
```

Comment:
Replace L34 - L38 with the below suggestion to fix the tests. I added a CFPBot GH token to Travis.

```suggestion
const opts = {
  uri:     'https://api.github.com/repos/cfpb/capital-framework/contents/packages',
  json:    true,
  headers: { 'user-agent': 'generator-cf' }
};
if ( process.env.GITHUB_PERSONAL_TOKEN ) {
  opts.headers['Authorization'] = `token ${process.env.GITHUB_PERSONAL_TOKEN}`
}
const components = requestPromise( opts ).then( filterComponents ).catch( console.error );
```